velox/dwio/common/tests/TypeTests.cpp

```diff
@@ -57,6 +75,7 @@ TEST(TestType, selectedType) {
 
   selected.assign(12, true);
   cutType = buildSelectedType(typeWithId, selector);
+  assertEqualTypeWithId(typeWithIdCopy, typeWithId);
```

Comment:
reverse the order of typeWithId and typeWithIdCopy, since actual is usually before expected in the parameter list